File: 1539-kth-missing-positive-number/1539-kth-missing-positive-number.java
class Solution {
    public int findKthPositive(int[] arr, int k) {
        return find(arr,k);
    }

    private static int find(int[] arr,int k){
        
        for(int i=0;i<arr.length;i++){
            int missing = arr[i]-(i+1);
            if(missing>=5){
                return k+i;
            }
        }
        return k + arr.length;
    }
}

File: 1539-kth-missing-positive-number/1539-kth-missing-positive-number.cpp
class Solution {
public:
    int findKthPositive(vector<int>& arr, int k) {
        int n=arr.size();
        for(int i=0;i<arr.size();i++){
            int missing=arr[i]-(i+1);
            if(missing>=k){
                return i+k;
            }

        }

        return n+k;
    }
};